static/bidder-info/ix.yaml

```diff
@@ -1,4 +1,3 @@
-disabled: true
```

Comment:
You must provide a default endpoint for an adapter to be enabled by default.

Reply:
@SyntaxNode Thanks for the review, discussed internally, will close this PR for now.